src/__template__/ComponentTemplate.vue

```diff
@@ -109,21 +153,15 @@ const rootElementStyles = computed((): Record<string, string> => ({
       viewBox="0 0 24 24"
       width="100%"
       xmlns="http://www.w3.org/2000/svg"
-    >
-      <title
-        v-if="title"
-        data-testid="kui-icon-svg-title"
-      >
-        {{ title }}
-      </title>
-      {%%ICON_SVG_INNER_HTML%%}
-    </svg>
+      v-html="svgProcessedContent"
+    />
   </component>
 </template>
 
 <style lang="scss" scoped>
```

Comment:
Can we get rid of the scope IDs in the HTML output if we remove the `scoped` attribute here? (`lang` is also unnecessary I think)